src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,768 @@
+	"messages.securitycraft:mrat.cantBind": "No puedes vincular una mina que no te pertenece.",
+	"messages.securitycraft:mrat.bound": "Mina en # vinculada al control remoto.",
+	"messages.securitycraft:mrat.unbound": "Mina en # desvinculada del control remoto.",
+	"messages.securitycraft:srat.noSlots": "¡No te queda espacio para vincular más torretas!",
+	"messages.securitycraft:srat.cantBind": "No puedes vincular una torreta que no te pertenece.",
+	"messages.securitycraft:srat.bound": "Torreta en # vinculada al control remoto.",
+	"messages.securitycraft:srat.unbound": "Torreta en # desvinculada del control remoto.",
+	"messages.securitycraft:universalBlockModifier.cantModify": "\"",
+	"messages.securitycraft:universalOwnerChanger.noName": "Tienes que darle el nombre del nuevo propietario a este bloque. Usa un yunque para hacerlo.",
+	"messages.securitycraft:universalOwnerChanger.cantChange": "Este bloque no puede tener propietario. Haz clic derecho en un bloque de SecurityCraft apropiado.",
+	"messages.securitycraft:universalOwnerChanger.notOwned": "Este bloque no es tuyo, no puedes cambiar su propietario.",
+	"messages.securitycraft:universalOwnerChanger.changed": "El propietario ahora es #.",
+	"messages.securitycraft:universalOwnerChanger.noBlockClaiming": "No puedes reclamar bloques sin propietario.",
+	"messages.securitycraft:irc.connected": "El chat de IRC no es compatible desde la v1.8.3 de SecurityCraft. ¡Si necesitas ayuda, dirígete al canal «#help» de SecurityCraft en Discord!",
+	"messages.securitycraft:cageTrap.captured": "#player ha sido atrapado en #location.",
+	"messages.securitycraft:frame.rightclick": "Haz clic derecho con un monitor vinculado para ver.",
+	"messages.securitycraft:command.sc.usage": "Usa: «/sc connect», «/sc disconnect», «/sc contact» O «/sc resume».",
+	"messages.securitycraft:naming.alreadyMatches": "Este nombre ya coincide con «#n».",
+	"messages.securitycraft:naming.error": "No puedes poner al bloque el nombre «#n», contiene el caracteres no válidos (#c).",
+	"messages.securitycraft:thanks": "¡Gracias por usar SecurityCraft, #!",
+	"messages.securitycraft:tip": "Consejo:",
+	"messages.securitycraft:tip.scHelp": "Escribe «/sc help» para ver el manual de SecurityCraft, que te mostrará ayudas sobre los bloques y objetos de SecurityCraft.",
+	"messages.securitycraft:tip.trello": "Echa un vistazo al tablón de Trello de SecurityCraft; ¡informa de errores o haznos sugerencias!",
+	"messages.securitycraft:tip.patreon": "¿Quieres apoyar a SecurityCraft?",
+	"messages.securitycraft:tip.discord": "¡Únete a nuestro Discord para chatear y recibir notificaciones de actualizaciones y anuncios de SecurityCraft!",
+	"messages.securitycraft:tip.scserver": "¡El servidor oficial de SecurityCraft ha vuelto! Únete en la IP: 64.38.249.237",
+	"messages.securitycraft:universalKeyChanger.notKeypad": "Haz clic derecho en un teclado con el cambiador de contraseñas universal para modificarla.",
```

Comment:
"Haz clic derecho en un teclado con el cambiador de contraseñas universal para modificar su contraseña"

Reply:
Changing it! (Applied the new name «Modificador de contraseñas universal» and changed to «cambiar su contraseña» to avoid redundancy.)